pkg/curatedpackages/reader.go

```diff
@@ -47,7 +49,7 @@ func (r *PackageReader) ReadImagesFromBundles(ctx context.Context, b *releasev1.
 	for _, vb := range b.Spec.VersionsBundles {
 		bundleURI, bundle, err := r.getBundle(ctx, vb)
 		if err != nil {
-			logger.Info("Warning: Failed getting bundle reference", "error", err)
+			logger.Info("Warning: Failed getting image reference", "error", err)
```

Comment:
nit: This error stems from a lack of a bundle and not an image.

Reply:
Fixed.